### PR TITLE
Fix/fix country and sc form fields

### PR DESCRIPTION
**Pull request by gabrielHosino**
#### What is the purpose of this pull request?

This PR makes some changes for the initial release of this integration. The first release will only be avaiable for Brazil and to the currency BRL so:

- We filter trade policies(sc) that do not have BRL as currency
- We change the countries response to only show Brazil as an option

#### How to test it?

You can use this [workspace](https://hosinotiktok--sandboxbrdevbrl.myvtex.com/admin/tiktok)

#### Screenshots or example usage
image

#### Types of changes

- [ ] Chore (non-breaking change which doesn't change any functionalities)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Review by anamaiabap — APPROVED** [review]

**Review by anamaiabap — COMMENTED** [review]
Tem algumas alterações de prettier e lint para serem corrigidos

File: react/components/form/CatalogIntegrationSettings.tsx [hunk 1]
```diff
@@ -10,6 +10,26 @@ export const CatalogIntegrationSettings = () => {
...
+  }, [optionsContext, intl])
+
+  // TODO: This is a workaround for the release of the TikTok integration, will change once the lauch is global
+  const isTradePolicySelectDisabled = useMemo(() => {
+    if (optionsContext.tradePolicies?.length === 0) {
+      return true
+    }
+
+    return false
+  }, [optionsContext])
```
Comment (pscruzzz) [inline react/components/form/CatalogIntegrationSettings.tsx:31]: Something that could be done here, is to have a "IS_LAUNCHED_GLOBALLY" const that is a boolean and have an early return on useMemo based on this variable!

This way you have a more programatic approach to this situation, even though later on you might delete it!

**Review by pscruzzz — APPROVED** [review]
LGTM

File: messages/en.json [hunk 2]
```diff
@@ -16,6 +16,9 @@
   "admin/tiktok.startup-form.validation-required": "Required.",
   "admin/tiktok.startup-form.validation-email": "Invalid Email.",
   "admin/tiktok.startup-form.validation-number": "Input must be a number.",
+  "admin/tiktok.startup-form.country.helper.text": "The integration only accepts Brazil as country",
```
Comment (felipefadul) [inline messages/en.json:19]:
```suggestion
  "admin/tiktok.startup-form.country.helper.text": "The integration only accepts Brazil as country.",
```

File: messages/pt.json [hunk 3]
```diff
@@ -16,6 +16,9 @@
   "admin/tiktok.startup-form.validation-required": "Campo obrigatório.",
   "admin/tiktok.startup-form.validation-email": "Email inválido.",
   "admin/tiktok.startup-form.validation-number": "Campo deve conter apenas números.",
+  "admin/tiktok.startup-form.country.helper.text": "Atualmente a integração aceita apenas o Brasil como país",
```
Comment (felipefadul) [inline messages/pt.json:19]:
```suggestion
  "admin/tiktok.startup-form.country.helper.text": "Atualmente a integração aceita apenas o Brasil como país.",
```

File: react/components/form/CatalogIntegrationSettings.tsx [hunk 4]
```diff
@@ -10,6 +10,26 @@ export const CatalogIntegrationSettings = () => {
   const intl = useIntl()
   const optionsContext = useContext(IntegrationOptionsContext)
 
+  // TODO: This is a workaround for the release of the TikTok integration, will change once the lauch is global
```
Comment (felipefadul) [inline react/components/form/CatalogIntegrationSettings.tsx:13]:
```suggestion
  // TODO: This is a workaround for the release of the TikTok integration, will change once the launch is global
```

File: react/components/form/CatalogIntegrationSettings.tsx [hunk 5]
```diff
@@ -10,6 +10,26 @@ export const CatalogIntegrationSettings = () => {
...
+    if (optionsContext.tradePolicies?.length === 0) {
+      return `${intl.formatMessage(
+        MESSAGES.startupForm.emptyTradePolicyHelperText
+      )} ${intl.formatMessage(MESSAGES.startupForm.tradePolicyHelperText)}`
+    }
+
+    return `${intl.formatMessage(MESSAGES.startupForm.tradePolicyHelperText)}`
+  }, [optionsContext, intl])
+
+  // TODO: This is a workaround for the release of the TikTok integration, will change once the lauch is global
```
Comment (felipefadul) [inline react/components/form/CatalogIntegrationSettings.tsx:24]:
```suggestion
  // TODO: This is a workaround for the release of the TikTok integration, will change once the launch is global
```

File: react/providers/IntegrationOptionsProvider.tsx [hunk 6]
```diff
@@ -13,6 +15,27 @@ const IntegrationOptionsProvider: FC = (props) => {
...
+      return {
+        ...result.externalData,
+        countries: [{ name: 'Brazil', code: 'BR', currencies: [] }],
+      }
+    }
+
+    return undefined
+  }, [result])
+
+  // TODO: This is a workaround for the release of the TikTok integration, will change once the lauch is global
```
Comment (felipefadul) [inline react/providers/IntegrationOptionsProvider.tsx:30]:
```suggestion
  // TODO: This is a workaround for the release of the TikTok integration, will change once the launch is global
```

**Review by felipefadul — APPROVED** [review]
LGTM! I've only pointed out some typos.

**Review by anamaiabap — APPROVED** [review]

File: react/components/form/CatalogIntegrationSettings.tsx (hunk 1 above)
Comment (gabrielHosino) [reply react/components/form/CatalogIntegrationSettings.tsx:31]: Hello! I think i got your point, but given the timeline we have right now and the fact that in the end these lines will be deleted i think it wont impact much

**Review by MIzidio — APPROVED** [review]

**Pull request merged**